### ISSUE TITLE
Error on starting node server 

### ISSUE THREAD

**Issue by RomainGuillaume** [angular/universal-starter#312]
Hello!

I have an application based on a regular Angular 2 structure, that I wish to convert into Angular Universal's.

After Long and painful typescript corrections (@ building stage), I get the following error on the `npm run start` command:
 
```
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /home/romain/Code/MetaNews/frontend-node/dist/**/* src/index.html
[nodemon] starting `node dist/server/index.js`
/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:20759
                            throw new Error("Unexpected " + _this._getTypeDescriptor(importedType) + " '" + __webpack_require__.i(__WEBPACK_IMPORTED_MODULE_4__facade_lang__["i" /* stringify */])(importedType) + "' imported by the module '" + __webpack_require__.i(__WEBPACK_IMPORTED_MODULE_4__facade_lang__["i" /* stringify */])(moduleType) + "'");
                            ^

Error: Unexpected value 'hooks' imported by the module 'MainModule'
    at /home/romain/Code/MetaNews/frontend-node/dist/server/index.js:20759:35
    at Array.forEach (native)
    at CompileMetadataResolver.getNgModuleMetadata (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:20744:44)
    at RuntimeCompiler._compileComponents (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:32361:47)
    at RuntimeCompiler._compileModuleAndComponents (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:32299:37)
    at RuntimeCompiler.compileModuleAsync (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:32290:21)
    at NodePlatform.cacheModuleFactory (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:56374:25)
    at Object.createEngine (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:43495:28)
    at Object.<anonymous> (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:59630:47)
    at Object.<anonymous> (/home/romain/Code/MetaNews/frontend-node/dist/server/index.js:59688:30)
[nodemon] app crashed - waiting for file changes before starting...
```
I have no clue about what this "hooks" function or variable stands for ? It's not in my application code anyway.

Tried to google it but nothing found yet.

Here is my **node.module.ts** :
```typescript
import { NgModule } from '@angular/core';
import { FormsModule } from '@angular/forms';
import { RouterModule } from '@angular/router';
import * as moment from 'moment';
import { UniversalModule, isBrowser, isNode } from 'angular2-universal/node'; // for AoT we need to manually split universal packages
import { LocalStorage } from './local-storage.ts';
import { NodeMetanews } from './node.metanews.ts';

// Will be merged into @angular/platform-browser in a later release
// see https://github.com/angular/angular/pull/12322
import { Meta } from './angular2-meta';

export function getLRU() {
  return new Map();
}
export function getRequest() {
  return Zone.current.get('req') || {};
}
export function getResponse() {
  return Zone.current.get('res') || {};
}

export const UNIVERSAL_KEY = 'UNIVERSAL_CACHE';

@NgModule({
  bootstrap: [ require('./templates/default/assets/js/components/app.component').AppComponent ],
  imports: [
    UniversalModule,

    FormsModule,
    moment,
    RouterModule.forRoot([], { useHash: false }),
    require('./templates/default/assets/js/components/app.module').AppModule,
  ],
  providers: [
    { provide: 'isBrowser', useValue: isBrowser },
    { provide: 'isNode', useValue: isNode },

    { provide: 'req', useFactory: getRequest },
    { provide: 'res', useFactory: getResponse },

    { provide: 'LRU', useFactory: getLRU, deps: [] },
    { provide: 'Metanews', useFactory: NodeMetanews },
    
    { provide: 'LocalStorage', useValue: LocalStorage },
    Meta
  ]
})
export class MainModule {}
```
It must be application related, but the error doesn't make it clear, any help will be appreciated !

Thanks for taking some time for reading this.
